e2e/gui/src/test/java/com/epam/pipeline/autotests/PlatformPreferencesTest.java

```diff
@@ -237,6 +237,52 @@ public void allowToSpecifyLustreFSTypeAndThoughput() {
         }
     }
 
+    @Test
```

Comment:
I suppose we should add @CloudProviderOnly(values = {Cloud.AWS}) annotation to the test

Reply:
Done